servers/cromwell/jobs/controllers/aggregation_controller.py

```diff
@@ -0,0 +1,21 @@
+import connexion
+from jobs.models.aggregation_response import AggregationResponse
+from datetime import date, datetime
+from typing import List, Dict
+from six import iteritems
+from ..util import deserialize_date, deserialize_datetime
+
+
+def get_job_aggregations(timeFrame, projectId=None):
+    """
+    Query for aggregated jobs in the given time frame.
+    
+    :param timeFrame: Time Frame to aggregate over.
+    :type timeFrame: str
+    :param projectId: The ID of the project to query.
+    :type projectId: str
+
+    :rtype: AggregationResponse
+    """
+    # return 'do some magic!'
+    raise NotImplemented('function not implemented yet.')
```

Comment:
This exception type isn't imported:
See for example this import https://github.com/DataBiosphere/job-manager/blob/master/servers/dsub/jobs/controllers/jobs_controller.py#L12